modules/appeals_api/config/schemas/hlr_post.json

```diff
@@ -0,0 +1,270 @@
+{
+  "$schema": "http://json-schema.org/draft-07/schema#",
+  "type": "object",
+  "properties": {
+    "data":     { "$ref": "#/definitions/data" },
+    "included": { "$ref": "#/definitions/included" }
+  },
+  "additionalProperties": false,
+  "required": ["data", "included"],
+  "definitions": {
+
```

Comment:
What's up with your linter? Two paragraph spacers seems unnecessary.

Reply:
Unnecessary, but made it much easier for me to read.

Reply:
I understand, just want to make sure we agree that this is how we want to have it organized and we stay consistent across all of our projects using this that's all.